x/ibc/07-tendermint/update.go

```diff
@@ -85,6 +85,8 @@ func checkValidity(
 	}
 
 	// Verify next header with the last header's validatorset as trusted validatorset
+	// TODO: Figure out a better way to add some leeway in currentTimestamp checking either in Tendermint or here
```

Comment:
Per @melekes it sounds like this has just been fixed upstream, so let's remove this and instead update upstream?